tests/utils/nodes/nodes.go

```diff
@@ -57,3 +65,26 @@ func GetNumOfReadyNodesInCluster(cs *client.ClientSet) (int32, error) {
 
 	return int32(numOfNodesExistsInCluster), nil
 }
+
+// CordonOrUnCordonNodeByName cordones/uncordones a node by a given node name.
+func CordonOrUnCordonNodeByName(nodeName string, clients *client.ClientSet, cordon bool) (bool, error) {
```

Comment:
As per my previous recommendation (no need to do it): You can have this code inside an internal function in this package called `setUnschedulableValue(clients *client.ClientSet, nodeName string, value bool) (bool, error)`, so it could be called from the exported `nodes.Cordon(string nodeName) (bool, error)` & `nodes.Uncordon(string nodeName) (bool, error)` ones.